apps/prairielearn/src/cron/autoFinishExams.ts

```diff
@@ -15,46 +15,41 @@ import { config } from '../lib/config';
  *
  * @see assessment.gradeAssessmentInstance
  */
-export function run(callback: (err?: Error | null) => void): void {
+export async function run() {
   const params = [config.autoFinishAgeMins];
-  sqldb.call('assessment_instances_select_for_auto_finish', params, function (err, result) {
-    if (ERR(err, callback)) return;
-    const examList = result.rows;
+  const result = await sqldb.callAsync('assessment_instances_select_for_auto_finish', params);
+  const examList = result.rows;
 
-    async.eachSeries(
-      examList,
-      function (examItem, callback) {
-        logger.verbose('autoFinishExams: finishing ' + examItem.assessment_instance_id, examItem);
-        // Grading was performed by the system.
-        const authn_user_id = null;
-        // Don't require the assessment to be open. This is important to
-        // ensure we correctly handle the case where the PrairieLearn process
-        // dies in the middle of grading a question. In that case, the assessment
-        // would have already been closed, but we still need to grade it.
-        const requireOpen = false;
-        // Override any submission or grading rate limits.
-        const overrideGradeRate = true;
-        // We don't have a client fingerprint ID, so pass null.
-        const clientFingerprintId = null;
-        assessment.gradeAssessmentInstance(
```

Comment:
This was the last usage of the non-async version of this function, so I dropped the `gradeAssessmentInstanceAsync` wrapper and made this function itself async 🎉